packages/harpguru-core/src/components/menu-of-tunings/menu-of-tunings.tsx

```diff
@@ -53,7 +69,18 @@ export const MenuOfTunings = (menuProps: MenuProps): React.ReactElement => {
     useGlobal,
   ])
   const zoomItemTapHandler = useCallback(
-    useDispatch(reduceForNewColumnBoundsByZoomId),
+    useDispatch(
+      (currentColumnBounds, activeDegreeMatrix, zoomId) =>
+        reduceForNewColumnBoundsByZoomId(
+          currentColumnBounds,
+          activeDegreeMatrix,
+          zoomId
+        ),
+      'columnBounds'
+    ),
+    // TODO: This and a great many other things should be based on the
```

Comment:
TODO here

Reply:
Not sure there are lots of examples actually. Think this might be the last one remaining.

Reply:
Raised as techdebt in https://github.com/js-jslog/harpguru/issues/120#issuecomment-938452028